### ISSUE TITLE
WinUI 3 ScrollViewer does not bubble PointerCaptureLost event when ancestor calls CapturePointer in PointerPressed

### ISSUE THREAD

**Issue by rozele** [microsoft/microsoft-ui-xaml#9883]
### Describe the bug

In UWP, if you have something like:
```
var stackPanel = new StackPanel();
var scrollViewer = new ScrollViewer();
stackPanel.Children.Add(scrollViewer);
stackPanel.PointerPressed += (sender, args) => {
  stackPanel.CapturePointer(args.Pointer);
}
stackPanel.PointerCaptureLost += (sender, args) => {
  Debug.WriteLine("PointerCaptureLost");
}
```
when you start a direct manipulation on the ScrollViewer with a touch gesture, the PointerCaptureLost event bubbles to the StackPanel. In WinUI 3, this does not occur.

This manifests in a limitation for React Native Windows, which uses pointer routed events at the top-level WinUI component to proxy events to JavaScript, and relies on ScrollViewer bubbling the PointerCaptureLost event to know when to cancel a gesture sequence emitted to JavaScript.

### Steps to reproduce the bug

1. Set up code like the above, where a ScrollViewer is nested in a WinUI component ancestor that calls `CapturePointer` in it's `PointerPressed` callback.
2. Subscribe the `PointerCaptureLost` event on the ancestor that captures the pointer.
3. Use a pan gesture to scroll the ScrollViewer
4. Observe that the PointerCaptureLost event does not bubble to the ancestor, nor does PointerReleased (expected) or PointerCanceled, so we have no terminal event to establish that a touch gesture has been interrupted / canceled.

### Expected behavior

WinUI 3 ScrollViewer should bubble a PointerCaptureLost event like it does in UWP when a touch gesture results in direct manipulation in a ScrollViewer.

### NuGet package version

WinUI 3 - Windows App SDK 1.5.5: 1.5.240627000

### Windows version

Windows 11 (22H2): Build 22621

**Comment by rozele** [comment]
What makes this an even bigger problem is that WinUI 3 also does not appear to bubble the `PointerReleased` event to the ancestor *unless* we call `CapturePointer` on it.

**Comment by codendone** [comment]
This was resolved as duplicate of an internal bug. Changing this to fixed. It looks like the fix is in 1.6-experimental2 and 1.6-preview1.
